common/src/stack/pylib/stack/exception.py

```diff
@@ -96,8 +96,10 @@ def __init__(self, cmd, arg=None):
 
 class ArgNotFound(ArgError):
 
-	def __init__(self, cmd, arg, argtype):
-		super(ArgNotFound, self).__init__(cmd, arg, 'is not a valid %s' % argtype)
+	def __init__(self, cmd, arg, argtype,params = None):
+		if params is None:
+			params = 'No parameters'
```

Comment:
Only append 'with parameters' if there are any.  Other Arg processors use this exception, too, and the (English-)syntax might look weird.